landscape.yml

```diff
@@ -3057,18 +3057,18 @@ landscape:
               clomonitor_name: etcd
               summary_personas: Architects, Platform Engineers, SREs
               summary_tags: distributed, key-value, consistent, datastore, reliability
-              summary_use_case: >
+              summary_use_case: >-
                 Etcd is a distributed, reliable key-value store for the most critical data of a distributed system.
 
                 By using etcd, developers can ensure that their applications have access to up-to-date configuration data, even as they scale up or down, and
                 can maintain consistency, fault tolerance and coordination across multiple instances of the application.
-              summary_business_use_case: >
+              summary_business_use_case: >-
+                Etcd can be used by businesses to manage the configuration data and coordination needs of their applications, ensuring consistent and reliable performance at scale.
+              summary_release_rate: Major or Minor release approximately every 2 years, and patch release every 1-3 months
+              summary_integrations: Due to etcd's flexible and extensible architecture it has been integrated as a data store for many projects such as Kubernetes, CoreDNS and Hashicorp Vault.
                 Etcd can be used by businesses to manage the configuration data and coordination needs of their applications, ensuring consistent and reliable
                 performance at scale.
               summary_release_rate: Every 1-3 months
```

Comment:
This seems redundant now.

Reply:
I removed it in my first commit https://github.com/cncf/landscape/pull/3128/commits/5f9283d26f17921a5ed30f6f093511facd465077, but it was added back in the second & third commits.